CHANGELOG.md

```diff
@@ -150,6 +150,7 @@
    * ADDED: Add Korean (`ko-KR`) locale [#5501](https://github.com/valhalla/valhalla/pull/5501)
    * UPGRADED: pybind11 from 2.11.1 to 3.0.1 [#5539](https://github.com/valhalla/valhalla/pull/5539)
    * CHANGED: Replace `oneof bool` to `bool` for default false options [#5541](https://github.com/valhalla/valhalla/pull/5541)
+   * ADDED: `checksum` to GraphTileHeader, a 64bit MD5 hash of the OSM PBFs[#5542](https://github.com/valhalla/valhalla/pull/5542)
```

Comment:
I don't like this Github change.. files in the repo root were previously shown first in the "Files changed" tab, IMO that's much more sensible

Reply:
in my view it very much looks like CHANGELOG is part of `/valhalla`

<img width="309" height="242" alt="Image" src="https://github.com/user-attachments/assets/9c520bb8-f381-4ab1-bea5-e904a0560679" />